project/src/main/java/ca/bookstore3005/project/controllers/BankAccountController.java

```diff
@@ -0,0 +1,16 @@
+package ca.bookstore3005.project.controllers;
+
+import java.util.List;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import ca.bookstore3005.project.models.BankAccount;
+import ca.bookstore3005.project.services.BankAccountService;
+
+@Controller
+public class BankAccountController {
```

Comment:
Can you remove all controllers? These will be made on a view page basis, not model based. Thanks!